frontend/src/App.tsx

```diff
@@ -19,12 +19,12 @@ function App() {
                 <AppHeader jurisdiction={`Demo STLT`}/>
                 <div className="display-flex flex-row height-full">
                     <div
-                        className="flex-3 padding-top-1 padding-left-1 padding-right-4 minw-15 usa-dark-background bg-primary-dark text-base-lightest display-flex flex-column flex-gap-1 maxw-30"
+                        className="flex-3 padding-top-1 padding-left-1 padding-right-4 minw-15 usa-dark-background bg-primary-dark text-base-lightest display-flex flex-column flex-gap-1 maxw-30 side-nav"
```

Comment:
nit: Do we need `maxw-30` on here if we're setting `max-width:182px` elsewhere?

Reply:
will fix this